src/modules/routing/services/RecipientService.ts

```diff
@@ -0,0 +1,255 @@
+import type { InboundMessageContext } from '../../../agent/models/InboundMessageContext'
+import type { ConnectionRecord } from '../../connections'
+import type { MediationStateChangedEvent, KeylistUpdatedEvent } from '../RoutingEvents'
+import type { MediationGrantMessage, MediationDenyMessage, KeylistUpdateResponseMessage } from '../messages'
+import type { Verkey } from 'indy-sdk'
+
+import { Lifecycle, scoped } from 'tsyringe'
+
+import { EventEmitter } from '../../../agent/EventEmitter'
+import { RoutingEventTypes } from '../RoutingEvents'
+import { KeylistUpdateAction, MediationRequestMessage } from '../messages'
+import { KeylistMessage } from '../messages/KeylistMessage'
+import { MediationRole, MediationState } from '../models'
+import { MediationRecord } from '../repository/MediationRecord'
+import { MediationRepository } from '../repository/MediationRepository'
+import { assertConnection } from '../services/RoutingService'
```

Comment:
Isn't it imported from the same folder?

```suggestion
import { assertConnection } from './RoutingService'
```